lib/view.dart

```diff
@@ -156,49 +156,109 @@ void renderChartOptions(bool isComparisonChecked, bool isNormalisationChecked) {
   var labelCol = getGridLabelCol()..innerText = 'Options';
   var optionsCol = getGridOptionsCol();
 
-  var comparisonWrapper = html.DivElement()
-    ..classes = ['form-check', 'form-check-inline'];
-  var comparisonOption = html.InputElement()
-    ..type = 'checkbox'
-    ..id = 'data-comparison-option'
-    ..classes = ['form-check-input']
-    ..checked = isComparisonChecked
-    ..onChange.listen((e) {
-      command(UIAction.toggleDataComparison,
-          ToggleOptionEnabledData((e.target as html.CheckboxInputElement).checked));
+  var comparisonCheckbox = _getCheckboxWithLabel(
+      'comparison-option', 'Compare data', isComparisonChecked,
+      (bool isChecked) {
+    command(UIAction.toggleDataComparison, ToggleOptionEnabledData(isChecked));
+  });
```

Comment:
nit: I think this should be indented so that line 162 is further in than the header of the function? something like:
```suggestion
        command(UIAction.toggleDataComparison, ToggleOptionEnabledData(isChecked));
      });
```

Reply:
Yes this and the other one i'm unable to override it from vscode prettier. Will track it and see what rule i can apply for this later. #11